frontend/auth/resetPasswordController.js

```diff
@@ -3,19 +3,28 @@
 (function () {
     const app = angular.module("app");
 
-    app.controller('ResetPasswordController', function (AuthService, $mdDialog) {
+    app.controller('ResetPasswordController', function (AuthService, $state, $scope) {
         var resetCtrl = this;
 
         resetCtrl.email = '';
-        resetCtrl.showConfirmedRest = false;
+        resetCtrl.emailSent = false;
 
-        resetCtrl.resetPassword = function resetPassword() {
-            AuthService.resetPassword(resetCtrl.email);
-            resetCtrl.showConfirmedRest = true;
+        resetCtrl.resetPassword = function () {
+            AuthService.resetPassword(resetCtrl.email)
+            .then(_ => {
+                resetCtrl.emailSent = true;
+                $scope.$apply();
+            }).catch(err => console.error(err));
```

Comment:
good to send feedback of the error. 
display some error message. something like "An error occurred, verify your email and try again"